docs/integrations/sources/github.md

```diff
@@ -95,7 +95,8 @@ Your token should have at least the `repo` scope. Depending on which streams you
 
 | Version | Date       | Pull Request | Subject                                                                                                      |
 |:--------|:-----------| :--- |:-------------------------------------------------------------------------------------------------------------|
-| 0.2.20  | 2022-02-16 | [10384](https://github.com/airbytehq/airbyte/pull/10384) | Add new stream `Deployments`, `ProjectColumns`, `PullRequestCommits`                                       |
+| 0.2.21  | 2022-03-05 | [10878](https://github.com/airbytehq/airbyte/pull/10878) | Fix error handling for 404 streams                                                                         |
```

Comment:
nit: Fix error handling for unavailable streams with `404` status code.

Reply:
good point, fixed